test.js

```diff
@@ -957,17 +957,22 @@ test('lineHeight returns line-height', t => {
   t.deepEqual(a, { lineHeight: 1.23 })
 })
 
-test('lineHeight returns a scalar style', t => {
+test('lineHeight returns responsive line-height', t => {
```

Comment:
Thanks for this! I'd prefer to keep the test that checks for the theme value as a separate one, like the other style functions. Would you mind updating this to include tests for both:
- `lineHeight returns a scalar style`
- `lineHeight returns responsive line-height`

Reply:
Yeah, that should be ok. I will include both test.